bluepysnap/_plotting.py

```diff
@@ -174,7 +174,7 @@ def _update_raster_properties():
                 ax.set_yticklabels(labels)
                 if len(labels) > 1:
                     ax.set_ylim(-0.5, len(labels) - 0.5)
-            ax.set_ylabel("{}".format(y_axis))
+            ax.set_ylabel(f"{y_axis}")
```

Comment:
According to the docstring `y_axis` is already a string (and it cannot be None because of the previous check), so it could be:
```suggestion
            ax.set_ylabel(y_axis)
```

Reply:
Nice catch!